packages/os/settings-applier.service

```diff
@@ -1,8 +1,7 @@
 [Unit]
 Description=Applies settings to create config files
-After=storewolf.service sundog.service early-boot-config.service apiserver.service
-Requires=apiserver.service
-Wants=storewolf.service sundog.service early-boot-config.service
```

Comment:
Why remove all of the other dependency information? `sundog` is not the only service that's needed for `settings-applier.service` to succeed.

Reply:
`sundog` requires `early-boot-config.service`, which itself requires `apiserver.service`. So by specifying just `sundog`, we get the exact same behavior.